backend/azureblob/azureblob.go

```diff
@@ -217,6 +218,17 @@ This option controls how often unused buffers will be removed from the pool.`,
 				encoder.EncodeDel |
 				encoder.EncodeBackSlash |
 				encoder.EncodeRightPeriod),
+		}, {
+			Name: "public_access",
+			Help: `Public access level of a container: blob, container or private(default).
+
+This sets the public access level of a container. Default is none that means only authorized user can get access to the container.
+Other options are "blob" and "container" where
+container: Specifies full public read access for container and blob data. Clients can enumerate blobs within the container via anonymous request,
+but cannot enumerate containers within the storage account.
+blob: Specifies public read access for blobs. Blob data within this container can be read via anonymous request, but container data is not available.
+Clients cannot enumerate blobs within the container via anonymous request.`,
+			Advanced: true,
```

Comment:
Maybe add `Default: azblob.PublicAccessNone` - might need to cast that to a string - not sure what type `azblob.PublicAccessNone` is...